### PR TITLE
Add two points to FAQ

### PR DESCRIPTION
**Pull request by ekozhura**
I added two more points to FAQ in both languages:
- why thursday
- how do we choose a location

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,4 +49,10 @@ There are two rules only: respect others and don't forget to pay for yourself in
 Just join us. Usually, we propose one-two topics before the meeting and someone takes role of speaker and prepares some material.
 However, noone likes to prepare and often we just extemporize. As an option, two or more topics may be discussed at the same time.
 
+## Why thursday?
+We spend friday nights with our families and our closest friends. Whereas thursday is almost a week end, and the evening is usually free.
+
```
Comment (AMashoshyna) [inline README.md]: ## Why Thursday?
We spend Friday nights with family and closest friends, whereas Thursday is almost weekend and the evening is usually free.

File: README.md [hunk 2]
```diff
@@ -43,4 +49,10 @@ There are two rules only: respect others and don't forget to pay for yourself in
...
+## How do we choose a location?
+A day before the meeting we start a poll in our telegram chat, and everyone can vote for one or more most appealing places. When thursday comes, we choose the most voted place and book here a table.
```
Comment (AMashoshyna) [inline README.md]: ## How Do We Choose a Location?
A day before the meeting we start a poll in our telegram chat, and everyone can vote for one or more most appealing places. When Thursday comes, we choose the most voted place and book a table.

**Comment by ekozhura** [comment]
@AMashoshyna done

File: README.md [hunk 3]
```diff
@@ -20,6 +20,12 @@ BeerJS Kyiv це регулярні зустрічі київської джав
 Просто приходьте. Зазвичай ми маємо одну-дві теми які хтось пропонує заздалегідь.
 Але дуже часто все проходить імпровізовано, або кілька груп окремо обговорюють якісь теми.
 
+## Чому четвер?
+Вечір п'ятниці ми звикли проводити з сім'єю та/або близькими друзями. Тоді як четвер - майже кінець тижня і у багатьох вечір зазвичай вільний.
+
+## Як обираємо заклад?
+Напередодні зустрічі кожен бажаючий може проголосувати за один або декілька закладів. Голосування проводимо у телеграм-чаті. У четвер підбиваємо підсумки голосування, бронюємо столик на зручний час та підтягуємося.
```
Comment (m1n0s) [inline README.md]: I think that's better to use `Telegram-чат`. Same about English.

Comment (ekozhura) [reply README.md]: seems legit

**Pull request merged**